gulpfile.js/validate.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {src} = require('gulp');
+const {join} = require('path');
+const gulpAmpValidator = require('gulp-amphtml-validator');
+
+/**
+ * Validates all pages build into /platform/pages.
+ */
+function validate() {
+  return src(join(__dirname, '../platform/pages/**/*.amp.html'))
```

Comment:
Could use `@lib/utils/project.paths.GROW_BUILD_DEST` to be prepared when the build destination sometime gets move to `dist`.

Reply:
good point - done